### PR TITLE
new member Apoorva

### PR DESCRIPTION
**Pull request by Apoorva2696**

### CODE REVIEW

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please add entry in `mapping.json`

Also, please follow the photo rules:
```
Images should be a square
Size of the image should be greater than 500px X 500px
The face should be visible, centered and identifiable
Face should cover at least 60% of the image
```

File: members/apoorva/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	"first_name": "Apoorva",
+	"last_name": "Srivastava",
+	"yoe": 1,
+	"company": "Publicis Sapient",
+	"designation": "Associate Experience Technology L2",
+	"img": "./img.png",
+	"github_id": "Apoorva2696",
+	"linkedin_id": "apoorva-srivastava",
+	"twitter_id": "apoorvasrivastava",
+	"instagram_id": ""
```
Comment (Rucha1499) [inline members/apoorva/data.json:12]: Line number 12 can be removed if you are not providing the value. Thank you!

**Review by Rucha1499 — CHANGES_REQUESTED** [review]

File: members/apoorva/data.json (hunk 1 above)
Comment (Apoorva2696) [reply members/apoorva/data.json]: Done

**Review by swarajpure — CHANGES_REQUESTED** [review]
Possible to have the face centered? If we merge this PR right now, the photo will get cropped in a circle shape and it might crop some part of your face. Hope you get my point 😅

**Review by Rucha1499 — APPROVED** [review]

**Review by swarajpure — APPROVED** [review]
Great! Thank you! 🎉 
Welcome to the members page 😄

**Pull request merged**